src/context-items/rename.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { ContextBuilder } from '../context-builder';
+import { Meta } from '../base';
+import { Rename } from './rename';
+
+it('the new path is identical to the old one', () => {
```

Comment:
nitpicking, but I usually like to make test names read nicely with the `it`.
For example:

```suggestion
it('does nothing when the new path is identical to the old one', () => {
```